mu-plugins/mu-loader.php

```diff
@@ -2,48 +2,40 @@
 /**
  * By default, WordPress will load all files in the mu-plugins directory and ignore all the directories inside.
  *
- * This file is a workaround which will load all the plugins in the mu-plugins directory which follow the naming
- * convention of `plugin-name/plugin-name.php`.
+ * This file is a workaround which will load all the plugins in the mu-plugins directory which have a root-level file
+ * containing a plugin header (@link https://developer.wordpress.org/plugins/plugin-basics/header-requirements/).
  */
 
 defined( 'ABSPATH' ) || exit;
 
 require_once ABSPATH . 'wp-admin/includes/plugin.php';
 
-foreach ( glob( WPMU_PLUGIN_DIR . '/*/*.php' ) as $t51_file ) {
-	if ( ! is_readable( $t51_file ) ) {
-		continue;
+foreach ( glob( WPMU_PLUGIN_DIR . '/*/*.php' ) as $bpd_mu_plugin_file ) {
```

Comment:
What does bpd stand for?

Reply:
Ah! It's the initials from [the `Build Processes Demo` repo](https://github.com/a8cteam51/build-processes-demo) that we worked on as part of the task force. It was too much work to rename everything, so we just went with it. The Team51 CLI searches for references to `bpd` and replaces them with the initials for the new project.

Reply:
That makes sense, thanks!